src/main/java/core/basesyntax/strategy/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
According to the checklist, there should not be an empty line at the beginning of the class implementation. Please remove the empty line after the package statement.